gapic-generator-java/src/main/java/com/google/api/generator/gapic/Generator.java

```diff
@@ -29,7 +30,8 @@ public static CodeGeneratorResponse generateGapic(CodeGeneratorRequest request)
     GapicContext context = Parser.parse(request);
     List<GapicClass> clazzes = Composer.composeServiceClasses(context);
     GapicPackageInfo packageInfo = Composer.composePackageInfo(context);
+    List<ReflectConfig> reflectConfig = Composer.composeNativeReflectConfig(context);
```

Comment:
Are we always going to generate the reflect configs for every client library? Do we want to consider a flag to turn it off if needed? Maybe it's a requirement but I'm not sure if every client library need GraalVM support, also the file could be huge for some client libraries.

Reply:
CC @meltsufin and @mpeddada1 for additional opinions and visibility.

> Are we always going to generate the reflect configs for every client library? Do we want to consider a flag to turn it off if needed? Maybe it's a requirement but I'm not sure if every client library need GraalVM support...

Providing the reflect-config as no impact on non-native builds, so the plan is to generate it for every client library. I'd hesitate to create a flag for disabling the generation until we have a use-case for it.

> ...also the file could be huge for some client libraries.

I don't think size should be a concern yet. Compute has one of the largest set of protos (just based off googleapis *.proto file sizes), and its reflect-config.json (in this PR's /test goldens) is 43KB -- and there's so much repeated text that it's going to compress in the jar really well... a simple zip takes it down to 1KB.